pages_bots/page.py

```
@@ -0,0 +1,88 @@
+"""
+from newapi.page import MainPage, NEW_API, CatDepth, CatDepthLogin
+from wikiapi_new.page import MainPage, NEW_API, CatDepth, CatDepthLogin
+
+Usage:
+
+from wikiapi_new.page import CatDepth, CatDepthLogin
+from newapi.page import CatDepth, CatDepthLogin
+# CatDepthLogin(sitecode="en", family="wikipedia")
+# cat_members = CatDepth(title, sitecode='en', family="wikipedia", depth=0, ns="all", nslist=[], without_lang="", with_lang="", tempyes=[])
```

Pylint (3.3.7)

[convention] Line too long (139/100)

(C0301)